src/core/src/main.rs

```diff
@@ -94,6 +95,15 @@ fn main() {
         .value_of("connect")
         .unwrap_or("tcp://localhost:5050");
 
+    // Read configuration file, i.e. TOML confiuration file
+    let config_file = matches.value_of("config")
+        .expect("Config file is not input, use -h for information");
+
+    let file_contents = read_file_as_string(config_file);
+    debug!("Read file contents: {}", file_contents);
+    let mut config: PoetConfig = toml_converter::from_str(file_contents.as_str())
+        .expect("Error reading toml config file");
```

Comment:
If few fields are missing in toml file, will this conversion result in assigning some default values to PoetConfig members or will it fail ?

Reply:
It fails. It should have all the configs.